scripts/blender/nerfstudio_addon_1_0.py

```diff
@@ -0,0 +1,404 @@
+bl_info = {
+    "name": "Nerfstudio Add-On",
+    "description": "Create a Nerfstudio JSON camera path from the Blender camera path or import a Nerfstudio camera path as a Blender camera to composite Blender renders over a NeRF background render",
+    "author": "Cyrus Vachha",
+    "version": (1, 0),
+    "blender": (3, 0, 1),
+    "category": "Nerfstudio",
+}
+
+import json
+from math import atan, degrees, radians, tan
+
+import bpy
+from mathutils import Matrix
+
+
+# create a JSON camera path from the Blender camera animation
+class CreateJSONCameraPath(bpy.types.Operator):
```

Comment:
Can you add docstrings to this file in the same way that the rest of the repo does. In addition to types. I'm happy to walk through some of the typing if you have questions.